src/ValueParsers/CalendarModelParser.php

```diff
@@ -30,11 +30,12 @@ protected function stringParse( $value ) {
 		switch ( $value ) {
 			case '':
 			case 'gregorian':
-				return TimeFormatter::CALENDAR_GREGORIAN;
+				return IsoTimestampParser::CALENDAR_GREGORIAN;
```

Comment:
As I mentioned elsewhere, I think they should not go there unless TimeValue itself uses them.